marklogic-data-hub-central/ui/src/config/tooltips.config.js

```diff
@@ -62,6 +62,9 @@ const NewMatchTooltips = {
     'sourceDatabase': 'The database where the input data is read from. Must be the same as the Matching Step *Source Database*.',
     'description': 'The description of this matching configuration.',	    'targetDatabase': 'The database where to store the processed data. Must be the same as the Matching Step *Source Database*.',
     'sourceQuery' : 'The collection or CTS query that selects the source data to process in this configuration.',
+    'thesaurusUri' : 'The path to the thesaurus used to determine synonyms',
```

Comment:
@ryanjdew Does it matter what database the thesaurus is in? As a user, that's the first question I have when reading this.